src/App.jsx

```diff
@@ -14,45 +15,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
+
+function getPreperedGoods(goods, { sortField, reverse }) {
+  const preperedGoods = goods.map((good, i) => (
+    {
+      name: good,
+      length: good.length,
+      id: i + 1,
+    }
+  ));
```

Comment:
You can set ids just ones outside this function.

Reply:
> You can set ids just ones outside this function.

I'll try to use uuid=)